static/app/components/feedback/widget/useFeedbackWidget.tsx

```diff
@@ -13,9 +13,10 @@ interface Props {
 
 export default function useFeedbackWidget({buttonRef, messagePlaceholder}: Props) {
   const config = useLegacyStore(ConfigStore);
-  const client = getClient();
-  // Note that this is only defined in environments where Feedback is enabled (getsentry)
-  const feedback = client?.getIntegrationByName?.<Feedback>('Feedback');
+  const feedback =
+    Sentry.getClient()?.getIntegrationByName<
+      ReturnType<typeof Sentry.feedbackIntegration>
+    >('Feedback');
```

Comment:
can we use the `client.getFeedback()` helper here instead of doing the types manually

```suggestion
   const feedback = Sentry.getClient()?.getFeedback();
```